modules/pxp-module-puppet

```diff
@@ -48,7 +64,8 @@ end
 
 def check_config_print(cli_arg, config)
   command_array = [config["puppet_bin"], "agent", "--configprint", cli_arg]
-  process_output = Puppet::Util::Execution.execute(command_array)
+  process_output = Puppet::Util::Execution.execute(command_array,
+                                                   {:custom_environment => get_env_fix_up()})
```

Comment:
yes:
https://github.com/puppetlabs/puppet/blob/4.4.2/lib/puppet/util/execution.rb#L297
and:
https://github.com/puppetlabs/puppet/blob/4.4.2/lib/puppet/util.rb#L102-L135